android/build.gradle

```diff
@@ -31,6 +31,15 @@ android {
         minSdkVersion 21
     }
 
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
+    }
```

Comment:
Are you sure these changes are really necessary 🤔 I can already build for flutter 3.29 with the main branch